### PR TITLE
Add feature to upload images by pasting from clipboard

### PR DESCRIPTION
**Pull request by morygonzalez**
Add feature to upload image by pasting from clipboard. ~~Please don't merge. Still in progress.~~

It's done.

### CODE REVIEW

File: public/admin/entries/index.haml [hunk 1]
```diff
@@ -5,20 +5,20 @@
...
-        %th= t("#{@name}.created_at")
-        %th{ colspan: 2 }
-      - @entries.each_with_index do |entry, i|
-        %tr{ class: (i + 1).odd? ? :odd : :even }
-          %td.title= link_to entry.title, url(entry.link)
-          %td= entry.user.nil? ? '' : entry.user.name
-          %td= l(entry.created_at, format: :long)
-          %td= link_to t('edit'), url(entry.edit_link), class: :button
-          %td= link_to t('delete'), url("/admin/#{@name.pluralize}/#{entry.id}"), class: 'button', confirm: t('are_you_sure'), method: :delete
+#table_wrapper
```
Comment (morygonzalez) [inline public/admin/entries/index.haml]: Just fixed indentation. It was broken.

File: public/admin/layout.haml [hunk 2]
```diff
@@ -6,12 +6,10 @@
     %meta{ 'http-equiv': 'Content-Script-Type', content: 'text/javascript' }
     %meta{ name: 'viewport', content: 'width=device-width' }
     %title== #{@site.title} - Lokka
-    %link{ rel: 'shortcut icon', href: url('/admin/favicon.ico') }
-    %link{ rel: 'stylesheet', href: url(asset_path('admin/css', 'style.css')), type: 'text/css' }
-    %link{ rel: 'stylesheet', href: 'https://cdn.quilljs.com/1.3.6/quill.snow.css', type: 'text/css' }
-    %script{ src: 'https://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js', type: 'text/javascript' }
-    %script{ src: url(asset_path('admin/js', 'jquery-ujs.js')), type: 'text/javascript' }
```
Comment (morygonzalez) [inline public/admin/layout.haml]: Bundle jquery-ujs via npm and package.json. So we can remove this.

**Comment by morygonzalez** [comment]
Does anyone have objection to this Pull Request?

File: public/admin/js/src/FileUploader.js [hunk 3]
```diff
@@ -101,15 +122,9 @@ class FileUploader {
       return;
     }
 
-    if (textarea.value.length === 0) {
```
Comment (morygonzalez) [inline public/admin/js/src/FileUploader.js]: Removed silly line break detection.

**Comment by komagata** [comment]
@morygonzalez Looks nice!

**Comment by morygonzalez** [comment]
@komagata Thank you. I'll merge this. If something goes wrong, please let me know.

**Pull request merged**